jsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "root/*": ["./src/*"],
```

Comment:
I find it weird to have `root` to refer to `src`. Why not just having `src` referring to... `src`? Find it easier to understand and follow.

Reply:
En effet, je suis plutôt d'accord 👍 
Modification faite ici:  https://github.com/cozy/mespapiers/pull/21